### DISCUSSION TITLE
how do you setup nodes for a vpn connection?

### DISCUSSION THREAD

**Discussion by izar3** [evilsocket/opensnitch#1070] [Q&A]
This: https://github.com/evilsocket/opensnitch/issues/924

I am using openVPN/nm-openvpn on manjaro and im trying to block all connections not going through the vpn connection ```source IP is 10.*.*.*```

The only node I have is in Nodes tab is unix:/local

Preferences > General > Address:
```
unix:///tmp/osui.sock
unix:///run/user/1000/opensnitch//osui.sock
unix:///var/run/user/1000/opensnitch//osui.sock
```
https://github.com/evilsocket/opensnitch/wiki/Nodes but im not sure how to add address with wildcards?

I also tried the FAQ: https://github.com/evilsocket/opensnitch/wiki/FAQs#why-is-wireguardmullvadetc-not-working-with-opensnitch

Debug invalid connections is enabled

```
mount -t debugfs none /sys/kernel/debug                                                     1 ✘
mount: /sys/kernel/debug: must be superuser to use mount.

dmesg(1) may have more information after failed mount system call.
sudo mount -t debugfs none /sys/kernel/debug                                               32 ✘
```
And still returns No such file or directory
```
sudo cat /sys/kernel/debug/tracing/kprobe_event                                               ✔
cat: /sys/kernel/debug/tracing/kprobe_event: No such file or directory
```
System:
```
KDE Manjaro: 6.1.62-1-MANJARO (64-bit)
~ OpenSnitch GUI - 1.6.4 ~
protobuf: 4.25.0 - grpc: 1.59.2

AUR: opensnitch-ebpf-module 1.6.4-1
```

Possibly related?: https://github.com/evilsocket/opensnitch/issues/1010

**Comment by gustavo-iniguez-goya** [comment]
Hi @izar3 ,

The rule to allow connections from 10.x.x.x has the check `[x] Priority rule` marked?
On the other hand, there should be also a rule to block everything, Probably just after the rule to allow 10.x.x.x.

Right now we don't support complex rules/behaviours, like "Enable rules AllowVPNSources and BlockEverything when iface wg0 exists".
There was an ancient idea to create group of rules. That way, it could be easier to apply "VPN rules" , "Uni rules", etc.
